staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -145,9 +146,17 @@ type GenericAPIServer struct {
 	preShutdownHooksCalled bool
 
 	// healthz checks
-	healthzLock    sync.Mutex
-	healthzChecks  []healthz.HealthzChecker
-	healthzCreated bool
+	healthzLock                sync.Mutex
```

Comment:
optional nit: we're up to five fields for health checks at this point, three are the same as readiness.  I would consider a struct.

Reply:
That sounds like a good idea. I'll refactor this in a follow-up PR.